contracts/DomainRegistry.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Does this contract covers all most probable questions in terms of data monitoring? -3

Reply:
- add DomainReleased and DomainCreated
- update Domain struct
- update tests

Reply:
Unfortunately, all this does not help much the requirement
still -3 points